cri/v1alpha2/cri_utils.go

```diff
@@ -847,21 +847,40 @@ func (c *CriManager) updateCreateConfig(createConfig *apitypes.ContainerCreateCo
 	return nil
 }
 
-func toCriContainerState(status apitypes.Status) runtime.ContainerState {
-	switch status {
-	case apitypes.StatusRunning:
-		return runtime.ContainerState_CONTAINER_RUNNING
-	case apitypes.StatusExited:
-		return runtime.ContainerState_CONTAINER_EXITED
+func toCriContainerState(state *apitypes.ContainerState) (criState runtime.ContainerState, reason string) {
```

Comment:
Why just define the return type `runtime.ContainerState, string` and return the value in the function?

Reply:
Did you mean the named return value?

Reply:
yes